ansible_collections/arista/avd/molecule/eos_cli_config_gen/documentation/devices/management-gnmi-grpc-tunnel.md

```diff
@@ -0,0 +1,82 @@
+# management-gnmi-grpc-tunnel
+
+## Table of Contents
+
+- [Management](#management)
+  - [Management Interfaces](#management-interfaces)
+  - [Management API GNMI](#management-api-gnmi)
+
+## Management
+
+### Management Interfaces
+
+#### Management Interfaces Summary
+
+##### IPv4
+
+| Management Interface | description | Type | VRF | IP Address | Gateway |
+| -------------------- | ----------- | ---- | --- | ---------- | ------- |
+| Management1 | oob_management | oob | MGMT | 10.73.255.122/24 | 10.73.255.2 |
+
+##### IPv6
+
+| Management Interface | description | Type | VRF | IPv6 Address | IPv6 Gateway |
+| -------------------- | ----------- | ---- | --- | ------------ | ------------ |
+| Management1 | oob_management | oob | MGMT | - | - |
+
+#### Management Interfaces Device Configuration
+
+```eos
+!
+interface Management1
+   description oob_management
+   vrf MGMT
+   ip address 10.73.255.122/24
+```
+
+### Management API GNMI
+
+#### Management API GNMI Summary
+
+| Transport | SSL Profile | VRF | Notification Timestamp | ACL |
+| --------- | ----------- | --- | ---------------------- | --- |
```

Comment:
Let's remove this empty table

Reply:
Addressed in [a6f7a88](https://github.com/aristanetworks/ansible-avd/pull/2696/commits/a6f7a88a1652f803ca160e92672127ddfbfa5b94)